prisma/schema.prisma

```diff
@@ -84,6 +84,8 @@ model Team {
 
   createdAt DateTime @default(now())
   updatedAt DateTime @updatedAt
+
+  inviteLink String? @unique // Add this line for the unique share link
```

Comment:
I think a more appropriate field name is `inviteCode`

Reply:
I think it makes more confusion in code.

![image](https://github.com/user-attachments/assets/a356d0c7-6224-4fe8-9806-b13848544600)

like above image.

we can use inviteCode on "updatedTeam" but in "res.json" inviteLink is suitable.

Reply:
If you want to change then tell me in which file I have to change. Because I am not too much comfortable with prisma. can you make it easy for me?